pallets/identity/src/lib.rs

```diff
@@ -601,13 +593,15 @@ decl_module! {
 
             // Also set current_did roles when acting as a signing key for target_did
             // Re-dispatch call - e.g. to asset::doSomething...
-            let new_origin = frame_system::RawOrigin::Signed(sender).into();
+            // let new_origin = frame_system::RawOrigin::Signed(sender).into();
```

Comment:
I am not sure if we are using this fn, but  commented lines disable its functionality.